gluon/scheduler.py

```diff
@@ -912,7 +923,7 @@ def report_task(self, task, task_report):
             d = dict(status=status,
                      next_run_time=task.next_run_time,
                      times_run=task.times_run,
-                     times_failed=0
+                     # times_failed=0
```

Comment:
I don't think you really want this commented out. If times_failed doesn't get reset after each successful run then it becomes a cumulative failure counter over the task's entire lifetime so over time even very rare failures could cause the times_failed to surpass the allotted retries and the task to incorrectly land in the FAILED state.